src/queue.hpp

```diff
@@ -175,6 +182,11 @@ struct cvk_command_queue : public _cl_command_queue,
     }
 
     cvk_buffer* get_or_create_printf_buffer() {
+        if (m_init_image_queue) {
+            cvk_error_fn("Could not get printf buffer from init-image queue");
+            CVK_ASSERT(false);
+            return nullptr;
+        }
```

Comment:
Why is this required?

Reply:
because the queue for image initialisation does not have a context, which is needed for that function.
Image initialisation should not need printf.